### PR TITLE
Add jake's line

### PR DESCRIPTION
**Pull request by LlamaBack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
 Cole & Jake
+
+new line by jake
+
```
Comment (coleanthony1990) [inline README.md:4]: comment goes here

**Review by coleanthony1990 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -3,3 +3,6 @@ Cole & Jake
 new line by jake
 
 hello
+
```
Comment (coleanthony1990) [inline README.md:6]: comment here

**Review by coleanthony1990 — CHANGES_REQUESTED** [review]
change something

**Review by coleanthony1990 — APPROVED** [review]

**Review by LlamaBack — COMMENTED** [review]
wassap

File: README.md [hunk 3]
```diff
@@ -1 +1,8 @@
 Cole & Jake
+
+new line by jake
+
```
Comment (coleanthony1990) [inline README.md:4]: hello good

**Pull request merged**